src/pymatgen/symmetry/groups.py

```diff
@@ -88,7 +109,10 @@ def is_supergroup(self, subgroup: SymmetryGroup) -> bool:
         Returns:
             bool: True if this group is a supergroup of the supplied group.
         """
-        warnings.warn("This is not fully functional. Only trivial subsets are tested right now. ")
+        warnings.warn(
+            "This is not fully functional. Only trivial subsets are tested right now. "
```

Comment:
typo: `Only trivial (subsets->supergroup)...`

maybe extract these duplicate messages into a `trivial_group_warning` format string with a `which_set` parameter that you fill with `subgroup`/`supergroup` in `def is_(sub|super)group`

Reply:
Hi, I'm not sure what you mean. A function like this?
```
def trivial_group_warning(subgroup: str, supergroup: str) -> str:
    return (
        f"This is not fully functional. Only trivial subsets are tested right now."
        f"This will not work if the crystallographic directions of subgroup "
        f"{subgroup} and supergroup {supergroup} are different."
    )
```
that is called in `is_(sub|super)group`?
```
warnings.warn(trivial_group_warning(subgroup=self.symbol, supergroup=supergroup.symbol))
```

Reply:
Hey @janosh can you maybe elaborate on your comment from Aug 2? Thanks!